src/components/ManageUsers/UserItem.jsx

```diff
@@ -10,10 +10,10 @@ function UserItem({ user }) {
   };
   const icon = getIcon();
   return (
-    <li className="useritem">
+    <li className="user-item">
       <img src={`/icons/${icon}`} />
-      <p className="useritem__email">{user.email}</p>
-      <button type="active" className="useritem_delete">
+      <p className="user-item__email">{user.email}</p>
+      <button type="submit" className="user-item_delete">
```

Comment:
this is button type="button" because it fires some action and has no form there https://html.com/attributes/button-type/